tests/integration/conftest.py

```diff
@@ -32,7 +32,7 @@
 from inspirehep.factory import create_app
 
 
-@pytest.yield_fixture(scope='session', autouse=True)
+@pytest.yield_fixture(scope='session')
```

Comment:
@jacquerie @mihaibivol OK if I kill this? Then the `db_only_app` works nicely below.